packages/tooltip/src/Tooltip.story.js

```diff
@@ -176,7 +177,7 @@ storiesOf("Tooltip/Tooltip", module)
           justifyContent: "space-between",
           width: "100%",
           position: "fixed",
-          bottom: 0,
+          bottom: 30,
```

Comment:
The bottom right button in this story is slightly out of bounds:
![image](https://user-images.githubusercontent.com/1459283/62223124-d0b6a400-b38a-11e9-9b2a-a2baf9b5c3c2.png)

I think it's caused by the `position: fixed`
We can remove it and use this on that `div` at line 152:
```jsx
<div
  style={{
      display: "flex",
      flexDirection: "column",
      justifyContent: "space-between",
      height: "calc(100vh - 30px)",
  }} 
>
```

Reply:
fixed, thanks